apps/ledger-live-desktop/src/renderer/components/TransactionConfirm/index.tsx

```diff
@@ -192,79 +193,88 @@ const TransactionConfirm = ({
+            {typeTransaction === "Approve" ? t("approve.description") : t("sign.description")}
+          </Text>
+          <Alert type="primary" mb={26}>
+            <Trans
+              i18nKey={
+                typeTransaction === "Approve"
+                  ? amountTransaction === "Unlimited MATIC"
+                    ? "approve.unlimited"
+                    : "approve.limited"
+                  : "sign.securityCheck"
+              }
+              values={{
+                recipientWording,
+              }}
+            />
+          </Alert>
+          <Box
+            style={{
+              width: "100%",
             }}
-          />
-        </Alert>
-        <Box
-          style={{
-            width: "100%",
-          }}
-          mb={20}
-        >
-          {fields.map((field, i) => {
-            const MaybeComponent = fieldComponents[field.type];
-            if (!MaybeComponent) {
-              console.log(
-                `TransactionConfirm field ${field.type} is not implemented! add a generic implementation in components/TransactionConfirm.js or inside families/*/TransactionConfirmFields.js`,
+            mb={20}
+          >
+            {fields.map((field, i) => {
+              const MaybeComponent = fieldComponents[field.type];
+              if (!MaybeComponent) {
+                console.log(
+                  `TransactionConfirm field ${field.type} is not implemented! add a generic implementation in components/TransactionConfirm.js or inside families/*/TransactionConfirmFields.js`,
+                );
+                return null;
+              }
+              return (
+                <MaybeComponent
+                  key={i}
+                  field={field}
+                  account={account}
+                  parentAccount={parentAccount}
+                  transaction={transaction}
+                  status={status}
+                />
               );
-              return null;
-            }
-            return (
-              <MaybeComponent
-                key={i}
-                field={field}
-                account={account}
-                parentAccount={parentAccount}
-                transaction={transaction}
-                status={status}
+            })}
+          </Box>
+        </Container>
+        {Footer && (
+          <>
+            <HorizontalSeparator />
+            <Footer transaction={transaction} />
+          </>
+        )}
+        {!Footer && manifestId && Object.keys(termsOfUse).includes(manifestId) && (
+          <>
+            <HorizontalSeparator />
+            <Text marginTop={30}>
+              <Trans
+                i18nKey="approve.termsAndConditions"
+                values={{ appName: manifestName }}
+                components={[
+                  <Link
+                    key={manifestId}
+                    onClick={() => openURL(termsOfUse[manifestId as keyof typeof termsOfUse])}
+                    to={"https://www.ledger.com"}
```

Comment:
it could be wise to make it a constant in `urls.ts` and maybe it's already an url there? are we sure we go to the / of ledger.com ?

Reply:
this `to` was unnecessary there